autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,6 +1,106 @@
+    meeting_count=0
+    for i in courseData['meetingsFaculty']:
+        parse_meeting(i,section_object,meeting_count)
+        meeting_count+=1
+
+def parse_meeting(meetingsData,section_object,meeting_count):
+    #gets values to create object
+    meeting_id = generate_meeting_id(str(section_object.section_num), str(meeting_count))
+    meeting_crn = meetingsData['meetingTime']['courseReferenceNumber']
+    meeting_building = meetingsData['meetingTime']['building']
+    #checks which days classes are on and adds them to attend_days
+    meeting_class_days = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+    attend_days = [False,False,False,False,False,False,False]
+    day_string = 'monday'
+    for j in range(0,len(meeting_class_days)):
+        day_string = meeting_class_days[j]
+        if meetingsData['meetingTime'][day_string] == True:
+            attend_days[j] = True
+    meeting_start_time = generate_meeting_time(meetingsData['meetingTime']['beginTime'])
+    meeting_end_time = generate_meeting_time(meetingsData['meetingTime']['endTime'])
+    meeting_class_type = meetingsData['meetingTime']['meetingType']
+    meeting_section = section_object
+    #creates and saves meeting object
+    meeting_object = Meeting(id=meeting_id, crn=meeting_crn, building=meeting_building, meeting_days=attend_days, start_time=meeting_start_time, end_time=meeting_end_time, meeting_type=meeting_class_type, section=meeting_section)
+    meeting_object.save()
+
+def parse_instructor(courseData):#should return instructor model
+    #gets values to create object
+    for i in courseData['faculty']:
+        instructor_id = i['displayName']#primary key. name is currently being used
+        instructor_email = i['emailAddress']
+        instructor_name = i['displayName']
+        #creates and saves instructor object
+        instructor_object = Instructor(id=instructor_id, email_address=instructor_email, name=instructor_name)
+        instructor_object.save()
+        return(instructor_object)
+
+def parse_course(courseData):
+    course_dept = courseData['subject']
+    course_number = courseData['courseNumber']
+    course_title = courseData['courseTitle']
+    course_term_code = courseData['term']
+    course_id = generate_course_id(course_dept, course_number, course_term_code)#primary key
+    course_desc = ''#long description-could not find in data
+    course_core_curriculum = ''#to be implented later
+    course_credit_hours = courseData['creditHourLow']
+    #creates and saves course object
+    course_object = Course(id=course_id, dept=course_dept, course_num=course_number, title=course_title, description=course_desc, core_curriculum=course_core_curriculum, credit_hours=course_credit_hours, term=course_term_code)
+    course_object.save()
+    #calls other parse fuctions
+    instructor_object = parse_instructor(courseData)
+    parse_section(courseData,instructor_object)
+    print(f'{courseData}\n')
```

Comment:
This print is fine for debugging, but should be removed before we merge this into `backend/master`